### ISSUE TITLE
Bug: transcription doesn't generate text file

### ISSUE THREAD

**Issue by glocalglocal** [ccbogel/QualCoder#402]
In Manage Files I can see my text files and one mp3 listed (all of them linked). I open the mp3 file to transcribe it. I immediately pasted transcription text I already had into the text box, and was planning to add more text. No text was saved. I expected (a) to see a new text file added to the list and (b) the text I typed to appear in the transcription box when I reopened the mp3 file in Manage Files. Neither of the two happened. Shouldn't (a) and (b) happen automatically as I type my text? 

Windows 10, QC 2.5. 

EDIT: There is probably something worse going on here. I now open the mp3 file to transcribe it and the text adding shortcuts don't work (eg ctrl+T). The legend reminding the user the shortcuts has disappeared -- see pic. I noticed all this after I installed QC 2.5. I recently upgraded to python-3.9.2. 

![trans](https://user-images.githubusercontent.com/16408198/113751250-4562e980-96fb-11eb-9f77-0ea4dbdfd7de.jpg)

**Comment by glocalglocal** [comment]
On further experimentation, this is what seems to have happened. I pasted transcribed text into the mp3 transcription box. I closed the window and opened the .transcribed entry from Manage Files but it was empty and date/time had not been updated. This happens every time I do the above. Now I know that if I opened .transcribed from Code Text, the pasted text would be there. 

I can't remember what I did then but when I re-opened the mp3 file, the text I had pasted was missing. I closed the mp3 file dialogue and deleted the .transcribed entry hoping that if I re-opened the mp3 file and pasted the text again, the .transcribed entry would be re-created. Now I know the .transcribed entry is created only once -- when the mp3 file is first linked. That in itself is a problem imv. 

With the .transcribed entry missing, I reopened the mp3 file. The hints line ('Transcription area:...')  was missing and Ctrl+T etc didn't work. 

I deleted the mp3 file and relinked it. The .transcribed entry was created. I opened the mp3 file, pasted the text again. The above problems persist (ie .transcribed appears empty, its date/time is not updated) but if I overlook them I can code the transcribed text in Code Text.

**Comment by ccbogel** [comment]
Thanks I will do some testing of this soon.

**Comment by ccbogel** [comment]
"I pasted transcribed text into the mp3 transcription box. I closed the window and opened the .transcribed entry from Manage Files but it was empty and date/time had not been updated. This happens every time I do the above. Now I know that if I opened .transcribed from Code Text, the pasted text would be there."

OK this is fixed. I update the text transcribed file before opening it. I was not doing that before.

**Comment by ccbogel** [comment]
"EDIT: There is probably something worse going on here. I now open the mp3 file to transcribe it and the text adding shortcuts don't work (eg ctrl+T). The legend reminding the user the shortcuts has disappeared -- see pic."
OK - providing the .transcribed file is present. This will happen if the .transcribed file has been coded at all.

**Comment by glocalglocal** [comment]
Does the date for .transcribed ever change? It always matches that of the mp3 here. 

If we can delete .transcribed, shouldn't there be a way to create it again? Otherwise you are stuck with an mp3 file you can't transcribe unless you unlink/delete and relink/import it again. Perhaps starting to transcribe should create .transcribed if it's not already there. 

Also, when we view .transcribed, shouldn't 'cancel' be greyed out as long as you haven't edited the text. Otherwise, OK and Cancel do the same thing.

**Comment by ccbogel** [comment]
If we can delete .transcribed, shouldn't there be a way to create it again?
Yes, Open Manage Files and create a text file. The file name must match the mp3 and add .transcribed at the end.
e.g. aaa.mp3    so create text file called aaa.mp3.transcribed

**Comment by ccbogel** [comment]
Does the date for .transcribed ever change? It always matches that of the mp3 here.
I dont change the date once created. It was how it was in RQDA, so I kept the dates as was, and didnt think more about it.

Also, when we view .transcribed, shouldn't 'cancel' be greyed out as long as you haven't edited the text. Otherwise, OK and Cancel do the same thing. - yes - will look in to it

**Comment by glocalglocal** [comment]
>Open Manage Files and create a text file. The file name must match the mp3 and add .transcribed at the end.

Or I can re-link the mp3. Both of these methods are not obvious, and can lead to errors (eg if the user mistypes filename.transcribed. I think every time an mp3 is opened, the .transcribed file should be created if it's not already there. After all, the whole point of adding an mp3 file is to transcribe it. Another possibility is not allowing the user to delete the .transcribed file as long as the mp3 file is there, but that might confuse things. 

>I dont change the date once created.

Unless it complicates things greatly, it would be good to keep it up to date. Displaying the creation date rather than the modified date was what made me think there was a problem with the file not being updated. If it stays this way, at least 'Date' should be 'Created'.

**Comment by ccbogel** [comment]
OK I have updated this, will close the issue. create a new issue with further specific problems.
